### PR TITLE
vmodel modifiers clarify example and diff 2.x v 3.x

### PR DESCRIPTION
**Pull request by marina-mosti**
Issue #342

### CODE REVIEW

File: src/guide/component-custom-events.md [hunk 1]
```diff
@@ -141,7 +141,13 @@ app.component('user-name', {
 
 ## Handling `v-model` modifiers
 
-In 2.x, we have hard-coded support for modifiers like `.trim` on component `v-model`. However, it would be more useful if the component can support custom modifiers. In 3.x, modifiers added to a component `v-model` will be provided to the component via the modelModifiers prop:
+In Vue 2.x, we had hard-coded support for modifiers like `.trim` or `.native` on component `v-model` bindings. And even though Vue 3.x also supports [out of the box modifiers](/guide/forms.html#modifiers), it would be more useful if our components could support custom modifiers of our own.
```
Comment (NataliaTepluhina) [inline src/guide/component-custom-events.md:144]: **suggestion**: I think we should remove 2.x mention here completely and just mention we have built-in modifiers with the link as nothing changes except `.native` modifier.

Also, let's remove the statement about custom modifiers being **more** useful (it really depends on the use-case). What about just `useful`?

```suggestion
As we already mentioned, `v-model` has [built-in modifiers](/guide/forms.html#modifiers) like `.trim` or `.native`. In some cases, you might also want to add your own custom modifier.
```

File: src/guide/component-custom-events.md [hunk 2]
```diff
@@ -141,7 +141,13 @@ app.component('user-name', {
 
 ## Handling `v-model` modifiers
 
-In 2.x, we have hard-coded support for modifiers like `.trim` on component `v-model`. However, it would be more useful if the component can support custom modifiers. In 3.x, modifiers added to a component `v-model` will be provided to the component via the modelModifiers prop:
+In Vue 2.x, we had hard-coded support for modifiers like `.trim` or `.native` on component `v-model` bindings. And even though Vue 3.x also supports [out of the box modifiers](/guide/forms.html#modifiers), it would be more useful if our components could support custom modifiers of our own. 
+
+Let's create an example custom modifier, `capitalize`, that capitalizes the first letter of the string provided by the `v-model` binding.
+
+In 3.x, modifiers added to a component `v-model` will be provided to the component via the `modelModifiers` prop. In the below example, we have created a component that contains a `modelModifiers` prop and defaults to an empty object.
```
Comment (NataliaTepluhina) [inline src/guide/component-custom-events.md:148]: **suggestion**: we are on 3.x docs already so we could remove version mention 😅 

```suggestion
Modifiers added to a component `v-model` will be provided to the component via the `modelModifiers` prop. In the below example, we have created a component that contains a `modelModifiers` prop that defaults to an empty object.
```

File: src/guide/component-custom-events.md [hunk 3]
```diff
@@ -141,7 +141,13 @@ app.component('user-name', {
...
+
+Notice that when the component's `created` lifecycle hook triggers, the `modelModifiers` prop contains `capitalize` and it's value is `true` - due to it being set on the `v-model` binding `v-model.capitalize="bar"`.
```
Comment (NataliaTepluhina) [inline src/guide/component-custom-events.md:150]:
```suggestion
Notice that when the component's `created` lifecycle hook triggers, the `modelModifiers` prop contains `capitalize` and its value is `true` - due to it being set on the `v-model` binding `v-model.capitalize="bar"`.
```

Comment (marina-mosti) [reply src/guide/component-custom-events.md:150]: oops 😅

**Comment by marina-mosti** [comment]
Thanks for the feedback @NataliaTepluhina 
I've gone ahead and made the changes, with a little rewording of the intro paragraph

File: src/guide/component-custom-events.md [hunk 4]
```diff
@@ -141,7 +141,13 @@ app.component('user-name', {
 
 ## Handling `v-model` modifiers
 
-In 2.x, we have hard-coded support for modifiers like `.trim` on component `v-model`. However, it would be more useful if the component can support custom modifiers. In 3.x, modifiers added to a component `v-model` will be provided to the component via the modelModifiers prop:
+When we were learning about form input bindings, we saw that `v-model` has [built-in modifiers](/guide/forms.html#modifiers) like `.trim` or `.native`. In some cases, however, you might also want to add your own custom modifiers.
```
Comment (NataliaTepluhina) [inline src/guide/component-custom-events.md:144]: My bad, I haven't caught it on the first round of review: `.native` does not exist anymore, so we should remove it

Comment (marina-mosti) [reply src/guide/component-custom-events.md]: Oops, yeah. Removed.

**Pull request merged**